_posts/en/meetings/2016-06-16-meeting.md

```diff
@@ -0,0 +1,116 @@
+---
+title: IRC meeting summary for 2016-06-16
+permalink: /en/meetings/2016/06/16/
+name: 2016-06-16-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2016-06-16/?msg=68050508&page=2)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-06-16-19.00.html)
+ 
+---
+ 
+## Main topics
+ 
+- Compactblock and feature freeze
+- Segwit update
+- Replacements with RBF in 0.13
+- GetBlockTemplate (GBT)
+
+## Compactblock and feature freeze
+
+### background
+ 
+[BIP152][]: “Compact block relay” is a proposed idea by BlueMatt for decreasing the bandwidth used during block relay by using short transaction IDs for transactions that should be in the mempool of the node. As a side-effect this also results in reducing the block transfer latency. Read the [compact blocks FAQ](https://bitcoincore.org/en/2016/06/07/compact-blocks-faq/) for more information.
```

Comment:
The pull request is done by BlueMatt, but the idea and general design is Gregory Maxwell iirc.